tincr/io/design/tincr_checkpoints.tcl

```diff
@@ -520,7 +521,7 @@ proc ::tincr::write_routing_rs2 {args} {
 			if {[llength $tiles] == 2} {
 				# assuming that the second tile in the tile list is the interconnect tile
 				set switchbox_tile [lindex $tiles 1]
-				set route_string [string range [get_property ROUTE $net] 3 end-3]
+				set route_string [string range [get_property ROUTE $net] 3 end-4]
```

Comment:
Add a comment explaining what this line of code is doing (perhaps even include an example)